src/locales.js

```diff
@@ -38,7 +38,13 @@ export const en_EN = {
   LBL_PANELITEM_REMOVE: 'Remove panelitem',
   LBL_PANEL_POSITION: 'Panel Position',
   LBL_SAVE_AND_LOG_OUT: 'Save Session & Log Out',
-  LBL_LOG_OUT: 'Log Out'
+  LBL_LOG_OUT: 'Log Out',
+  
```

Comment:
I've added all of these strings to the latest `@osjs/client` release. Could you change the usage ?

https://github.com/os-js/osjs-client/commit/09a1170273ca50e72a791151cdc42d53bb50c71c